### ISSUE TITLE
Using something else then require.main.filename

### ISSUE THREAD

**Issue by rudovjan** [WisePricer/quadro#197]
```js
static async appDir() {
   return this.getPackageRootDir(require.main.filename)
}
```

Because we expect that we run / test our apps using
`
node app.js
`

---

But if we run let say 
`
mocha ./node_modules/quadro/lib/test/independent_test_loader.js
`

require.main.file is mocha directory

Do we have ideas how to make it universal?

**Comment by rudovjan** [comment]
Solved by
```js
require.main.filename = require.main.filename.replace('/node_modules/mocha/bin/_mocha', '')
```

**Comment by igorshapiro** [comment]
Resolved by #195
